org.o20.word.appdata.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ideally this file should be maintained in your git repository and installed by cmake.